recipes/hangar/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "hangar" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 85bbb1e8fe0cac56ca86257283a96472bd8410443d1b819d0e6b67f0a0d63214
+
+build:
+  number: 1
```

Start from 0, it only needs to changed if the previous build was deployed:
```suggestion
  number: 0
```

---

fixed!